pkg/kube/constants.go

```diff
@@ -244,8 +244,20 @@ const (
 
 	AUTH = "auth"
 
-	Region = "Region"
+	// KubeProvider stores the kubernetes provider used to create the cluster or install
+	KubeProvider = "kubeProvider"
```

Comment:
Should we remove in future the cloud provider from team settings? I think is more a cluster config.

Reply:
agreed